migrations/versions/7937dae319b5_.py

```diff
@@ -0,0 +1,65 @@
+"""empty message
+
+Revision ID: 7937dae319b5
+Revises: ba778ef9c615
+Create Date: 2020-09-21 17:17:10.542429
+
+"""
+from alembic import op
+import requests
+
+
+# revision identifiers, used by Alembic.
+revision = "7937dae319b5"
+down_revision = "ba778ef9c615"
+branch_labels = None
+depends_on = None
+nominatim_base_url = "https://nominatim.openstreetmap.org"
+nominatim_data_format = "&format=jsonv2&accept-language=en"
+
+
+def upgrade():
+    conn = op.get_bind()
+    # fetch existing country names
+    fetch_countries = (
+        "select distinct(unnest(country)) from projects where country is not null;"
+    )
+    countries = conn.execute(fetch_countries)
+    for country in countries:
+        country = country[0]
+        # search by name
+        url = nominatim_base_url + "/search?country=" + country + nominatim_data_format
```

Comment:
If we add `namedetails=1` param to that request we don't need to call the reverse geocoding endpoint, as it will return all the names 😃

Reply:
That was handy - I couldn't spot this initially while going through the nominatim API. Using it to avoid reverse geocoding